libavogadro/src/extensions/networkfetchextension.cpp

```diff
@@ -184,8 +187,22 @@ namespace Avogadro
     return redirectUrl;
   }
 
+  void NetworkFetchExtension::printSslErrors(QNetworkReply*, 
+                                             const QList<QSslError> &errors)
+  {
+    foreach(const QSslError &error, errors) {
+      qDebug() << QString("SSL Error: %1").arg(error.errorString());
+    }
+  }
+
   void NetworkFetchExtension::replyFinished(QNetworkReply *reply)
   {
+    // Print error messages
+    if (reply->error() != QNetworkReply::NoError) {
+      qDebug() << QString("Network Error: %1").arg(reply->errorString());
```

Comment:
tr() .. might also want to add a few other possible network error cases (e.g., server is down) to help with debugging.